pkg/process/checks/process_common_test.go

```diff
@@ -21,6 +21,18 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/procutil"
 )
 
+// Returns chunking and no chunking RunOptions
+func getChunkingOption(noChunking bool) *RunOptions {
+	return &RunOptions{RunStandard: true, NoChunking: noChunking}
+}
+
+//nolint:revive // TODO(PROC) Fix revive linter
```

Comment:
So don't copy/paste any of these `//nolint:revive // TODO(PROC) Fix revive linter`. I believe we switched to a different linter at some point and these methods didn't have comments, so they added these comments so the linter ignores them. I don't think the linter would complain about no comments for an function that's not exported, but I'm not sure.